extensions/simctl/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "simctl",
+  "title": "Simulator Control",
+  "description": "Control apple simulator devices",
+  "icon": "command-icon.png",
+  "author": "noppefoxwolf",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Simctl",
```

Comment:
Similar comment here, but maybe having `Simulator Control` as the `title` might be clearer. no